src/MahApps.Metro/Controls/Dialogs/BaseMetroDialog.cs

```diff
@@ -103,12 +103,13 @@ public double DialogMessageFontSize
         }
 
         /// <summary>Identifies the <see cref="DialogButtonFontSize"/> dependency property.</summary>
-        public static readonly DependencyProperty DialogButtonFontSizeProperty = DependencyProperty.Register(nameof(DialogButtonFontSize), typeof(double), typeof(BaseMetroDialog), new PropertyMetadata(12D));
+        public static readonly DependencyProperty DialogButtonFontSizeProperty = DependencyProperty.Register(nameof(DialogButtonFontSize), typeof(double), typeof(BaseMetroDialog), new PropertyMetadata(SystemFonts.MessageFontSize));
 
         /// <summary>
         /// Gets or sets the font size of any dialog buttons.
         /// </summary>
-        public double DialogButtonFontSize {
+        public double DialogButtonFontSize
+        {
```

Comment:
Maybe it's worth checking in some editorconfigs and StyleCop configs. This kind of work must be adding up with this much traffic on this repo 😄

Reply:
Yeah, I should add it to the existing editorconfig.